com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -456,6 +459,26 @@ internal set
                 }
             }
 
+            /// <summary>
+            /// Returns whether this state update was an axial frame synchronization
+            /// (only applies when UseUnreliableDeltas is enabled)
+            /// </summary>            
+            public bool IsAxisFrameSync()
```

Comment:
This method seems odd to me. It's a boolean method that simply returns a boolean property and has a name that is almost, but not quite, the same as the property. Since the property it returns is internal, could we just rename it and make it public?